ubuntu-advantage.1

```diff
@@ -75,6 +75,48 @@ Possible values are: active, inactive, or n/a
 Show version of the Ubuntu Advantage package.
 
 
+.SH CONFIGURATION
+By default, ubuntu-advantage client configuration options are read from
+(/etc/ubuntu-advantage/uaclient.conf). If a (uaclient.conf) is present in
+the current working directory it will be used instead of the default conf file.
+
+The following configuration options are available:
+.TP
+.B
+\fBcontract_url\fP
+The ubuntu advantage contract server URL. (\fBhttps://contacts.canonical.com\fP)
```

Comment:
Ubuntu Advantage

Can we also remove the defaults from each option? One less thing to update when we change things.